src/lib/api/voting.ts

```diff
@@ -1,35 +1,32 @@
-import qs from 'qs';
-
 import { client } from '../axios';
 import { VotingInfo } from './../../types/voting';
 
 export const postVoteData = (votingData: VotingInfo, token?: string) => client.post('/vote', votingData);
 
-export interface VoteInfo {
-  voteId: number;
-  title: string;
-  voteThumbnail: string;
-  createdAt: Date;
-  totalVoteCount: number;
-}
-
-export interface Result {
-  result: VoteInfo[];
-}
-
 export interface VoteListData {
   status: number;
   success: boolean;
   message: string;
   data: Result;
+}
+
+export interface Result {
+  result: VoteInfo[];
   resCursorId: number;
 }
 
+export interface VoteInfo {
+  voteId: number;
+  title: string;
+  voteThumbnail: string;
+  createdAt: Date;
+  totalVoteCount: number;
+}
+
 export const getCurrentVoteData = async (resCursorId: number) => {
-  console.log(resCursorId);
   try {
-    const data = await client.get<VoteListData>(`vote/getCurrentVote/${resCursorId}`);
-    return data.data.data.result;
+    const data = await client.get<VoteListData>(`/vote/getCurrentVote/${resCursorId}`);
+    return data.data.data;
   } catch (err) {
     console.log('실패');
     // console.error(err);
```

Comment:
```suggestion
export const getCurrentVoteData = async (resCursorId: number) => {
  try {
    const data = await client.get<AxiosResponse<VoteResultData>>(`vote/getCurrentVote/${resCursorId}`);
    return data.data;
  } catch (err) {
    console.error(err);
  }
};
```

여기서 AxiosResponse사용하면 굳이 VoteListData 정의할 필요없어요!
```typescript
export interface AxiosResponse<T = any, D = any> {
  data: T;
  status: number;
  statusText: string;
  headers: RawAxiosResponseHeaders | AxiosResponseHeaders;
  config: AxiosRequestConfig<D>;
  request?: any;
}

```

Reply:
오오 AxiosResponse를 만들어줬구낭!!! 리팩토링때 바아로 반영하겠습니당 ㅎㅎ